SWLOR.Game.Server/Feature/StatusEffectDefinition/FlamethrowerStatusEffect.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.CombatService;
+using SWLOR.Game.Server.Service.SkillService;
+using SWLOR.Game.Server.Service.StatusEffectService;
+using SWLOR.Game.Server.Service.AbilityService;
+
+namespace SWLOR.Game.Server.Feature.StatusEffectDefinition
+{
+    public class FlameThrowerEffectDefinition : IStatusEffectListDefinition
```

Comment:
This should be renamed to FlameThrowerStatusEffectDefinition to be consistent. Additionally, your class names should always match the actual file name. Please rename it from FlamethrowerStatusEffect.cs to FlameThrowerStatusEffectDefinition.cs